templates/zerver/help/archive-a-stream.md

```diff
@@ -1,34 +1,34 @@
-# Delete a stream
+# Archive a stream
 
 {!admin-only.md!}
 
-Deleting a stream will immediately unsubscribe all users from the stream,
+Archiving a stream will immediately unsubscribe all users from the stream,
 remove the stream from search and other typeaheads, and remove the stream's
 messages from **All messages**.
 
-Deleting a stream does not delete a stream's messages. Users will still be
+Archiving a stream does not delete a stream's messages. Users will still be
 able to find any given message by searching for it. However, links to
 messages and topics in the stream may or may not continue to work.
 
 In most cases, we recommend [renaming streams](/help/rename-a-stream) rather
-than deleting them.
+than archiving them.
 
-### Delete a stream
+### Archive a stream
 
 {start_tabs}
 
 {relative|stream|all}
 
 1. Select a stream.
 
-1. On the right, click **Delete**.
+1. On the right, click **Archive**.
 
-1. Click **Yes, delete this stream**.
+1. Click **Yes, archive this stream**.
 
 {end_tabs}
 
 !!! warn ""
-    Deleting a stream is irreversible.
+    Archiving a stream is irreversible.
```

Comment:
This should be a separate commit, but we should perhaps change the text here to "Archiving a stream is currently irreversible, but we plan to change this behavior in future Zulip releases."

Reply:
(Not sure if there's something else we should add similar language, e.g. in the confirmation modal).

Reply:
Added to the doc for now. Seems like the right place. Putting information about our plans for future releases in that kind of modal in the app itself seems like somewhat strange UX potentially and also clutter :thinking: Users can always go to the doc if they're interested in more details